plover/gui/config.py

```diff
@@ -44,6 +44,7 @@
 SPACE_PLACEMENT_BEFORE = "Before Output"
 SPACE_PLACEMENT_AFTER = "After Output"
 SPACE_PLACEMENTS = [SPACE_PLACEMENT_BEFORE, SPACE_PLACEMENT_AFTER]
+TRANSLATION_TRANSPARENCY_LABEL = "Add Translations Dialog Transparency"
```

Comment:
I'm not a fan of this label, because it reads funny. I would prefer "Translation Dialog Transparency", after a bit of thought. What do you think?